### DISCUSSION TITLE
Three different variants of MBD

### DISCUSSION THREAD

**Discussion by mhyeok1** [libmbd/libmbd#51] [Q&A] [answered]
Dear Developer,

Firstly, I would like to express my sincere gratitude for your dedication and effort in creating such a wonderful job.

I have a question related to the MBD variants in pyMBD. 
It appears that there are three variants within MBD: plain, scs, and rsscs. 
and I was hoping you could elaborate on the distinctions between these three variants. plus the citations for each method.

I am looking forward to your answers.

Best Regards,
Minhyeok Lee

**Comment by jhrmnn** [comment, marked as answer]
Hi, and sorry for the very late answer.
- `plain` refers to the bare MBD Hamiltonian with no screening as in the experimental [MBD@TS]((http://scitation.aip.org/content/aip/journal/jcp/140/18/10.1063/1.4865104) method.
- `scs` refers to the MBD Hamiltonian preceded by the full-range screening, as in the older [MBD@SCS](http://link.aps.org/doi/10.1103/PhysRevLett.108.236402) method.
- `rsscs` refers to the MBD Hamiltonian preceded by the short-range screening, as in the [MBD@rsSCS](http://scitation.aip.org/content/aip/journal/jcp/140/18/10.1063/1.4865104) method.

You’re right to point out that this should be better documented, so I’ll create an issue out of this discussion.
